extras/classification/play.py

```diff
@@ -0,0 +1,27 @@
+from pathlib import Path
+
+from qibocal.fitting.classifier import (
+    linear_svm,
+    naive_bayes,
+    plots,
+    random_forest,
+    rbf_svm,
+    run,
```

Comment:
Since you converted to strings, in principle now you should be able to avoid the imports here
```suggestion
    plots,
    run,
```